samples-generator/bin/3d-tiles-samples-generator.js

```diff
@@ -506,7 +507,7 @@ function createBatchedWithVertexColors() {
     return saveBatchedTileset('BatchedWithVertexColors', tileOptions);
 }
 
-function createBatchedWithDataURI() {
+function createBatchedWithContentDataUri() {
     return saveBatchedTileset('BatchedWithContentUri', undefined, {
```

Comment:
Make sure to fix the name of the tileset itself to `BatchedWithContentDataUri` as well.